src/AbstractRepository.php

```diff
@@ -52,4 +54,29 @@ protected function sanitizePath($path)
 
         return Path::canonicalize($path);
     }
+
+    /**
+     * @param string $path
+     *
+     * @return PuliResource
+     */
+    protected function followLinksOrFail($path)
+    {
+        $resolvedWithLinks = $this->followLinks($path);
```

Comment:
If I understand correctly, this method only follows one level of links, no? I.e., if a link points to another link (hypothetically), this method will return the second link, not the final target?